__tests__/app.test.js

```diff
@@ -52,4 +52,53 @@ describe("NC News", () => {
       });
     });
   });
+
+  describe('/api/articles/:article_id', () => {
+    test('GET 200: Responds with the article with the specified article_id.', () => {
+      return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(1);
```

Comment:
We know what this article should look like, so i would have tested specific values rather than hasOwn and type of below
EG expect(article.author).toBe(<the expected value>)